pkg/config/config.go

```diff
@@ -104,6 +106,12 @@ func NewWithViper(v *viper.Viper) (ServerConfig, error) {
 				return time.ParseDuration(input)
 			}
 
+			if in.String() == "string" && out.String() == "[]string" {
```

Comment:
Add tests for this

Reply:
Shouldn't this test capture this already? https://github.com/zapier/kubechecks/pull/320/files#diff-29cea1b5b831c8655c7155f43e2367cec73e66d1e338f8b3c7877a2f339b8811

Reply:
yeah. I added this comment before I saw the test 👍🏿